specification/dnsresolver/resource-manager/Microsoft.Network/preview/2020-04-01-preview/examples/DnsResolver_ListBySubscription.json

```diff
@@ -0,0 +1,64 @@
+{
+  "parameters": {
+    "subscriptionId": "abdd4249-9f34-4cc6-8e42-c2e32110603e",
+    "resourceGroupName": "sampleResourceGroup",
```

Comment:
resourceGroupName should not be present in ListBySubscription calls

Reply:
Addressed as a part of the latest commit.